lib/ClangImporter/ImportDecl.cpp

```diff
@@ -7903,9 +7903,39 @@ createUnavailableDecl(Identifier name, DeclContext *dc, Type type,
   return var;
 }
 
+static ExtensionDecl *figureOutTheDeclarationContextToImportInto(Decl* D, DeclContext *&DC, IterableDeclContext *&IDC) {
+  if (auto nominal = dyn_cast<NominalTypeDecl>(D)) {
+    DC = nominal;
+    IDC = nominal;
+    return nullptr;
+  }
+  else {
+    ExtensionDecl *ext = cast<ExtensionDecl>(D);
+    DC = ext;
+    IDC = ext;
+    return ext;
+  }
+}
+
+
+static void loadMembersOfBaseImportedFromClang(ExtensionDecl *ext) {
+  const NominalTypeDecl *base = ext->getExtendedType()->getAnyNominal();
+  auto *clangBase = base->getClangDecl();
+  if (!clangBase)
+    return;
+  base->loadAllMembers();
+  
+  // Sanity check: make sure we don't jump over to a category /while/
+  // loading the original class's members. Right now we only check if this
+  // happens on the first member.
+  if (auto *clangContainer = dyn_cast<clang::ObjCContainerDecl>(clangBase)){
```

Comment:
It looks like you are missing a space before the '{' at the end of the line here. Rather than worrying about this, you can use git-clang-format which will take care of all of these sorts of minor issues. If you want, I can show you how to set it up.